catalog/dags/data_refresh/data_refresh_task_factory.py

```diff
@@ -181,6 +190,13 @@ def create_data_refresh_task_group(
         # running against an index that is already promoted in production.
         tasks.append(create_filtered_index)
 
+        enable_alarms = PythonOperator(
+            task_id="enable_sensitive_cloudwatch_alarms",
+            python_callable=cloudwatch.enable_or_disable_alarms,
+            op_args=[True],
```

Comment:
Same note as the above:
```suggestion
            op_kwargs={
                "enable": True,
            },
```